russstation/code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -27,4 +27,16 @@
 		M.fakevomit(0)
 	return ..()
 
+
+/datum/reagent/consumable/superlaughter/traitor_pen_variant
+	name = "Super Laughter"
+	description = "Funny until you're the one laughing."
+	metabolization_rate = 1.5 * REAGENTS_METABOLISM
+	color = "#FF4DD2"
+	taste_description = "laughter"
+
+/datum/reagent/consumable/superlaughter/on_mob_life(mob/living/carbon/M)
+	M.emote("laugh")
+	..()
```

Comment:
```suggestion
/datum/reagent/consumable/superlaughter/traitor_pen_variant/on_mob_life(mob/living/carbon/M)
	M.emote("laugh")
	..()
```
Though fun fact about DM - if you define a proc a second time (like the superlaughter's on_mob_life), it treats the initial declaration as the parent - So what you did here is make it so the original superlaughter chemical makes you emote laugh

Reply:
Oops, I fixed it.